app/repositories/UserRepository.scala

```diff
@@ -18,7 +18,7 @@ class UserRepository  @Inject()(mongoComponent: MongoComponent)(implicit ec: Exe
 		IndexModel(ascending("crn"), IndexOptions().unique(true)
 		))
 ){
-	def create(user: User): Future[Boolean] = ???
+  def create(user: User): Future[Boolean] = collection.insertOne(user).toFuture().map(_ => true).recover { case _ => false }
```

Comment:
.map(_ => true) needs to have a failure case or else if it isn't created (because UUID already exists or any other reason) you will still return true.